### DISCUSSION TITLE
what happened to the preview program?

### DISCUSSION THREAD

**Discussion by Promenak** [community/community#19377] [Copilot Conversations] [answered]
I was a part of the preview program and as far as I remember there was supposed to be 2 versions of copilot a free one and a paid one with more features
but apparently the preview program is wiped and I can no longer use the copilot
that is disappointing to be honest because they didn't say their plan is to remove the free version and have a paid only copilot
I hope they do what they said and provide a free to use version

**Comment by melvincwng** [comment, marked as answer]
@Un-known-User as of 21st June 2022, Github Copilot is made generally available for all developers to use.

If you are a verified student or popular open source maintainer, it would be free to use. Otherwise, the rest would have a free 60 day trial to use Github Copilot.

Hope this clarifies.

More information can be found [here](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/)

If this reply is helpful, please kindly 'Mark as answer'. Thanks!
